docs/tutorial.rst

```diff
@@ -6,7 +6,30 @@ During GSOC 2021 a bunch of new tutorials were programmed by Hrithik Kumar Verma
 a `public share <https://fz-juelich.sciebo.de/s/IzNGi24Ki68iV7t#pdfviewer>`_
 
 
-Some videos about the Mission Support System (MSS)
+Get familiar by some videos about the Mission Support System (MSS).
+
+
+  .. video:: _static/mp4/tutorial_waypoints.mp4
+     :nocontrols:
+     :autoplay:
```

Comment:
I usually dislike autoplay, I prefer manually starting videos. As long as they don't have sound it might be fine though...

Reply:
it is the only one which gives there a little dynamic in creating a flight path. At the moment we have sound it gets disabled.